static/espeasy_default.css

```diff
@@ -124,21 +124,12 @@ h6 {
 	font-size: 10pt
 }
 
-code,
-kbd,
-pre,
-samp,
-tt,
-xmp {
-	font: 1em monospace, monospace
```

Comment:
i removed this because i couldn’t find where any of this is used and if so it is not the worst to not have monospace for it...

Reply:
Doesn't that apply to the Rules editor without the CodeMirror extension? There it would be nice to have a monospace font (though CodeMirror also doesn't have that). (Didn't check it though)

Reply:
the rule window gets its style from the textarea and therefore still has monospace as font

Reply:
System variables page.
```c++
void addSysVar_html(const String& input) {
  html_TR_TD();
  {
    addHtml(F("<pre>")); // Make monospaced (<tt> tag?)
    addHtml(F("<xmp>")); // Make sure HTML code is escaped. Tag depricated??
    addHtml(input);
    addHtml(F("</xmp>"));
    addHtml(F("</pre>"));
  }
  html_TD();
  addSysVar_html_parsed(input, false); // Not URL encoded
  html_TD();
  addSysVar_html_parsed(input, true); // URL encoded
  delay(0);
}
```

Reply:
Also the Level plugin uses the `tt` tag

Reply:
ok..i re-add `tt`  anything else?

Reply:
The code I showed actually uses the `xmp` tag.
Not sure if the `tt` is obsolete now.

Reply:
ok..`xmp` and `pre` where the only tags that came up in a text search...
and both are in the sys vars page you showed.... 
so these i keep..